ProcessLib/HeatTransportBHE/CreateHeatTransportBHEProcess.cpp

```diff
@@ -147,17 +144,13 @@ std::unique_ptr<Process> createHeatTransportBHEProcess(
         MaterialPropertyLib::createMaterialSpatialDistributionMap(media, mesh);
 
     // find if bhe uses python boundary condition
-    auto const isUsingPythonBC =
+    auto const is_using_python_bc =
         visit([](auto const& bhe) { return bhe.usePythonBC; }, bhes[0]);
-    if (isUsingPythonBC)
-    {
-        has_network_python_bc = true;
-    }
 
     //! Python object computing BC values.
     BHEInflowPythonBoundaryConditionPythonSideInterface* py_object = nullptr;
     // create a pythonBoundaryCondition object
-    if (has_network_python_bc)
+    if (is_using_python_bc)
```

Comment:
Minor; I'd have named it `using_python_bcs`. Then it reads like an English sentence in the if-condition.

Reply:
Thank you for the suggestion. Have been renamed in 1c511a0027102b9e1599687926abccbd6c63c9b1.